signac_dashboard/modules/navigator.py

```diff
@@ -0,0 +1,99 @@
+from flask import render_template, url_for
+
+from signac_dashboard.module import Module
+from signac_dashboard.util import escape_truncated_values
+
+
+class Navigator(Module):
+    """Displays links to jobs differing in one state point parameter.
+
+    This module may not update if the signac project changes because the module
+    caches the project schema when the signac-dashboard launches.
+
+    :param context: Supports :code:`'JobContext`
+    :type context: str
+    """
+
+    _supported_contexts = {"JobContext"}
+
+    def __init__(
+        self,
+        name="Navigator",
+        context="JobContext",
+        template="cards/navigator.html",
+        **kwargs,
+    ):
+        super().__init__(name=name, context=context, template=template, **kwargs)
+
+    def get_cards(self, job):
+        project = self._dashboard.project
+
+        def link_label(key, other_val):
+            """Returns the url and label for the job with job.sp[key] == other_val."""
+
+            similar_sp = job.sp()  # modifiable
+            similar_sp.update({key: other_val})
+
+            # Look only for exact matches of only changing one parameter
+            # in case of heterogeneous schema
+            other_job = project.open_job(similar_sp)
+            if other_job in project:
+                link = url_for("show_job", jobid=other_job.id)
+                label = other_job.sp[key]
```

Comment:
Is this equivalent? I think you can avoid the lookup if you know the value already.
```suggestion
                label = other_val
```

Reply:
Good catch